crates/bevy_ui/src/render/ui.wgsl

```diff
@@ -93,6 +93,14 @@ fn sd_rounded_box(point: vec2<f32>, size: vec2<f32>, corner_radii: vec4<f32>) ->
     return l + m - radius;
 }
 
+fn is_inside_corner(point: vec2<f32>, size: vec2<f32>, corner_radii: vec4<f32>) -> bool {
```

Comment:
If keeping this, add comment that it's currently available for debugging and not used for anything.

Reply:
Yep not going to keep it.